x/incentive/testutil/integration.go

```diff
@@ -214,12 +258,12 @@ func (suite *IntegrationTester) BalanceInEpsilon(address sdk.AccAddress, expecte
 	}
 }
 
-func (suite *IntegrationTester) VestingPeriodsEqual(address sdk.AccAddress, expectedPeriods vesting.Periods) {
+func (suite *IntegrationTester) VestingPeriodsEqual(address sdk.AccAddress, expectedPeriods vestingtypes.Periods) {
 	acc := suite.App.GetAccountKeeper().GetAccount(suite.Ctx, address)
 	suite.Require().NotNil(acc, "expected vesting account not to be nil")
-	vacc, ok := acc.(*vesting.PeriodicVestingAccount)
+	vacc, ok := acc.(*vestingtypes.PeriodicVestingAccount)
 	suite.Require().True(ok, "expected vesting account to be type PeriodicVestingAccount")
-	suite.Equal(expectedPeriods, vacc.VestingPeriods)
+	suite.ElementsMatch(expectedPeriods, vacc.VestingPeriods)
```

Comment:
There were type mismatches that caused testify equal to fail, eg `[]vestingtypes.Period` vs `vestingtypes.Periods`. I'll try updating it to use the same type so the order is also checked by just using `[]vestingtypes.Period` instead of slice alias.  Not sure if there's some way with testify to consider type aliases the same.